json/classic/src/test/input/json/jsonLayout.xml

```diff
@@ -0,0 +1,26 @@
+<!--
+
+    Copyright (C) 2016, The logback-contrib developers. All rights reserved.
+
+    This program and the accompanying materials are dual-licensed under
+    either the terms of the Eclipse Public License v1.0 as published by
+    the Eclipse Foundation
+
+      or (per the licensee's choosing)
+
+    under the terms of the GNU Lesser General Public License version 2.1
+    as published by the Free Software Foundation.
+
+-->
+<configuration>
+
+    <appender name="STR_LIST" class="ch.qos.logback.core.read.ListAppender">
+        <encoder class="ch.qos.logback.core.encoder.LayoutWrappingEncoder">
+            <layout class="ch.qos.logback.contrib.json.classic.JsonLayout">
+                <jsonFormatter class="ch.qos.logback.contrib.jackson.JacksonJsonFormatter"/>
+            </layout>
+        </encoder>
+    </appender>
+
+    <appender-ref ref="STR_LIST"/>
```

Comment:
Removing it does however produce a NPE at 
`        listAppender.doAppend(event);`